README.md

```diff
@@ -279,6 +284,81 @@ We have a small collection of algorithms, one for every day of the month. Scroll
   - [More on the Caesar Cipher](http://practicalcryptography.com/ciphers/caesar-cipher)
 
 
+### **December 15 - Pascal's triangle**
+- **Problem**
+  - The pascal’s triangle is given as:
+  ```
+	                    1
+                  1          1
+             1          2        1
+         1         3         3      1
+    1       4          6         4     1
+  ```
+ - Up To any number of rows. The task here is to generate a pascal’s triangle for n rows and using this find `(x+y)^n`
+- **Example**
+```
+Sample input:
+rows = 3
+Sample output:
+1       --> row 0
+11      --> row 1
+121     --> row 2
+1331    --> row 3
+Polynomial: (1x^3y^0)+(3x^2y^1)+(3x^1y^2)+(1x^0y^3)
+```
+- **Uses**
+  -  The numbers in Pascal's triangle have another name: Binomial coefficient, and they show up a bunch. They're generally written like `(nx)(nx)` where `n` is the row and `x` is the position (starting from 0, not one.)
+
+  - There is a formula for finding them, but for smaller numbers it's about as easy, or easier, to use the triangle as it is the formula.
+- **Application**
```

Comment:
Each of the applications are at the same indent level as the Application heading itself. Add a little indentation. 

![image](https://user-images.githubusercontent.com/10437649/49506976-7c176380-f8a5-11e8-82a8-0ba041b309bd.png)